contributing.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to {{Project-Name}}
+
+## Where to get help
+The main way to get help is on our [discord server](https://discord.gg/uh69TdKfBD).  Though, this guide should help you get started. It may be slightly lengthy, but it designed for those who are new so please don't let length intimidate you.
+
+## Code of Conduct
+Please be nice to each other when interacting with others and follow our [code of conduct](<link to code of conduct>).
+
+## How to contribute
+In order to contribute you can either add a new issue or you can create a pull request and link it to an existing issue. If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.
```

Comment:
> In order to contribute you can either add a new issue or you can create a pull request and ~~link it to~~ reference it in an existing issue.

Possible slight rewording

> If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.

We may want to encourage folks to create an issue before they start writing any code. The concern is that if someone does a bunch of work without discussing it first, and their pull request is declined, they may feel bad about the experience. Instead, if there is a discussion in an issue, they can get a sense for whether their work will be accepted.

Probably less of a concern for fixing bugs, but it's still a good idea to discuss it a bit first.